src/c++/vdo/base/message-stats.c

```diff
@@ -0,0 +1,1216 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * Copyright 2023 Red Hat
+ *
+ */
+
```

Comment:
Please tighten up the header comment by removing the blank line on line 4. It should look like the header message in other vdo/base .c files.